_board/ndgarage_ndbit6_v2.md

```diff
@@ -9,6 +9,8 @@ board_image: "unknown.jpg"
 downloads_display: false
 date_added: 2020-09-1
 family: atmel-samd
+bootloader_id: ndbit7
```

Comment:
Is this supposed to be `ncbit6`?

Reply:
I don't believe so. There was already an ndbit6 board (different than this) and there is nothing called ndbit7, so I figured they were being clever.

Reply:
Fair enough!